docs/reference/indices/analyze.asciidoc

```diff
@@ -86,6 +86,70 @@ curl -XGET 'localhost:9200/test/_analyze' -d '
 Will cause the analysis to happen based on the analyzer configured in the
 mapping for `obj1.field1` (and if not, the default index analyzer).
 
+If you want to get more advanced details, set `detail` to `true` (defaults to `false`). It will output all token attributes for each token.
```

Comment:
Might be good to put this in it's own section? Also should we mark this experimental?